examples/removeLiquidity/removeLiquidityNested.V3.ts

```diff
@@ -0,0 +1,112 @@
+/**
+ * Example showing how to remove liquidity from a V3 nested pool.
+ * V3 nested removes are always proportional.
+ * User must approve composite router to spend BPT token - See tests in test/v3/removeLiquidityNested for full flow.
+ * (Runs against a local Anvil fork)
+ * Note - remove transaction is not fully run in this example because we cannot use slot method to artificially give test account BPT balance.
```

Comment:
Should be solve-able when we get around to the major refactor of examples

Reply:
Yeah, exactly. Didn't want to get too deep into that at this stage.